oshi-core/src/main/java/oshi/util/GlobalConfig.java

```diff
@@ -41,9 +44,21 @@
     private static final Properties configuration = new Properties();
 
     static {
-        try (InputStream in = GlobalConfig.class.getResourceAsStream("/default.properties")) {
-            if (in != null) {
-                configuration.load(in);
+        // Load the configuration file from the classpath
+        try {
+            List<URL> resources = Collections.list(GlobalConfig.class.getClassLoader().getResources("oshi.properties"));
```

Comment:
Per the [`Class#getClassLoader`](https://docs.oracle.com/javase/6/docs/api/java/lang/Class.html#getClassLoader())  javadoc:
> Some implementations may use null to represent the bootstrap class loader. This method will return null in such implementations if this class was loaded by the bootstrap class loader.

A brief search suggests we prefer `Thread.currentThread().getContextClassLoader()` but I'm not sure if that allows a relative path?

Reply:
Good catch. I think it will still work with the context classloader, but I'll check.